src/lib/Hydra.pm

```diff
@@ -8,7 +8,6 @@ use Hydra::Plugin;
 use Hydra::Model::DB;
 use Catalyst::Runtime '5.70';
 use Catalyst qw/ConfigLoader
-                Unicode::Encoding
```

Comment:
The warning notes that we should define some encoding config -- uh, what should that look like?

Reply:
I think we're good: https://metacpan.org/pod/Catalyst#ENCODING